Flashcards.iamnikitakostin/FlashCards/FlashCards/Controllers/ConsoleController.cs

```diff
@@ -1,89 +1,87 @@
 ﻿using Spectre.Console;
 
-namespace FlashCards.Controllers
+namespace FlashCards.Controllers;
```

Comment:
🟢 **File-Scoped Namespaces**

💡 Although, remember to leave a blank line between namespace and class